simpa/Cargo.toml

```diff
@@ -40,3 +40,8 @@ tokio = { workspace = true, features = ["rt", "macros", "rt-multi-thread"] }
 [features]
 heap = ["dhat", "kaspa-alloc/heap"]
 semaphore-trace = ["kaspa-utils/semaphore-trace"]
+
+[profile.heap]
+inherits = "release"
+debug = true
+strip = false
```

Comment:
`cargo run simpa` now prints the following:
```
warning: profiles for the non root package will be ignored, specify profiles at the workspace root:
package:   ~/rusty-kaspa/simpa/Cargo.toml
workspace: ~/rusty-kaspa/Cargo.toml
```

and indeed `cargo run --profile heap --bin simpa --` works without it as well